app/views/templates/_help_accordion.html.erb

```diff
@@ -0,0 +1,16 @@
+          <div class="panel panel-default">
```

Comment:
Sure, all comments done in https://gist.github.com/2ae30894d7a8157507650aa5e696e1ea